internal/compiler/emitter.go

```diff
@@ -323,8 +340,19 @@ func sourceFileMayBeEmitted(sourceFile *ast.SourceFile, host printer.EmitHost, f
 		return true
 	}
 
-	// !!! Should JSON input files be emitted
-	return false
```

Comment:
```
// Json file is not emitted if outDir is not specified
    if (!options.outDir) return false;
```

this one is missing ?

Reply:
Good catch, I missed that one. Probably misread it as `outFile`